setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+packages = [
+    'robottelo',
+]
+
+requires = [
+    'ddt',
+    'fauxfactory',
+    'nailgun',
+    'paramiko',
+    'python-bugzilla',
+    'requests',
+    'selenium',
+]
+
+with open('README.rst', 'r') as f:
+    readme = f.read()
+
+setup(
+    name='robottelo',
+    version='0.1.0',
+    description='Robottelo is a test suite which exercises The Foreman.',
+    long_description=readme,
+    author=u'Satellite QE Team',
+    url='https://github.com/SatelliteQE/robottelo',
+    packages=packages,
```

Comment:
The standard `setuptools` provides a function that is capable of automatically finding the packages that you probably want to have distributed. It's used like so:

```python
from setuptools import find_packages, setup  # prefer setuptools over distutils

setup(
    packages=find_packages(),
    …
)
```

This does mean that we're depending entirely on setuptools and not on distutils. But is that a bad thing? I suspect setuptools is extremely widely supported, and using it has at least this one big advantage.